src/coldfront_plugin_api/scim_v2/groups.py

```diff
@@ -117,3 +134,26 @@ def _set_user_status_on_allocation(allocation, user, status):
                 status=AllocationUserStatusChoice.objects.get(name=status)
             )
         return au
+    
+    @staticmethod
+    def _set_user_on_project(project, user, status, role, enable_notifications):
+        pu = ProjectUser.objects.filter(
+            project=project,
+            user=user
+        ).first() # Why first, can't we assume this is always unique?
```

Comment:
That's a good question, but unless you are answering it here, remove the comment.